src/openrct2/paint/Paint.cpp

```diff
@@ -730,7 +730,7 @@ paint_session * paint_session_alloc(rct_drawpixelinfo * dpi)
     return session;
 }
 
-void paint_session_free(paint_session * session)
+void paint_session_free([[maybe_unused]] paint_session* session)
```

Comment:
For arguments like `session` here, that clearly just lack implementation, I'd rather leave them unmarked, so they can be easier spotted – especially without any `TODO` or issue filed for why it is marked as `[[maybe_unused]]`

Reply:
How can they be easier spotted without being marked? I believe it's quite the opposite. If someone touches this function, they see the attribute and realize it's not used yet. The current implementation of paint sessiosn doesn't need it, but another future implementation may.